Exercises.playground/Pages/Exercise3.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,17 @@
+//: [Previous](@previous)
+
+import Foundation
+
+func printIfPositiveInteger(number: String) {
```

Comment:
In this was asked to implement this in 2 different ways, 1 with compounded guards (the one implemented) and 1 with 2 guards.